content/rust/concepts/loops/loops.md

```diff
@@ -0,0 +1,93 @@
+---
+Title: 'Loops'
+Description: 'A loop is a control structure that allows you to repeatedly execute a block of code.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Loops'
+CatalogContent:
+  - 'Learn-rust'
+  - 'paths/computer-science'
+---
+
+A `loop` is a control structure that allows you to execute a block of code repeatedly. Loops are used to perform tasks that need to be done multiple times, such as iterating through a list of items, performing calculations, or waiting for a specific condition to be met.There typically three main types `for`, `while` and `loop`.
+
```

Comment:
```suggestion
## Syntax

```
We always add syntax heading then we add the example heading. We can make the different kinds of loops as sub headings.